rust/api/src/kv_store_tests.rs

```diff
@@ -428,34 +428,33 @@ pub trait KvStoreTestSuite {
 		let total_kv_objects = 20;
 		let page_size = 5;
 		for i in 0..total_kv_objects {
-			ctx.put_objects(Some(i as i64), vec![kv(&format!("{}k", i), "k1v1", 0)]).await?;
+			ctx.put_objects(None, vec![kv(&format!("{}k", i), "k1v1", 0)]).await?;
```

Comment:
Can you explain why you need to change any of the test cases at all? Shouldn't they all just pass when run against the in-memory store?